### DISCUSSION TITLE
Using MDA for ellipsoidal particles in LAMMPS and the new additional_columns option.

### DISCUSSION THREAD

**Discussion by jacob-votava** [MDAnalysis/mdanalysis#4708] [General]
Hello everyone! I am rehashing a conversation that happened with Hugo on the discord channel, since it would be beneficial for people to know if they're simulating ellipsoidal particles with LAMMPS and if they (like me) are new to MDA.

Currently MDanalysis does not natively support atom_style ellipsoid. However, in the most recent develop branch (2.8.0.dev0) there is now support for flexible column types. (Thank you developers!) For example, using a lammpsdump that's consistent with what works with visualizing ellipsoids in this documentation from [OVITO](https://www.ovito.org/manual/advanced_topics/aspherical_particles.html) the following MDAnalysis script can access the different quaternions/sizes

```python
traj = "npt.dump"
t = "isotropic_NPT_T1.0_P1.0.data"
atom_style = 'id type x y z c_q[1] c_q[2] c_q[3] c_q[4] c_diameter[1] c_diameter[2] c_diameter[3]'
u = mda.Universe(t, traj, format='LAMMPSDUMP', atom_style=atom_style, additional_columns=True)

for ts in u.trajectory[::10]:
    print(ts.data.keys())
```

This is available in the most recent develop branch, but not yet on main. Hopefully it's helpful to someone in the future!

**Comment by hmacdope** [comment]
Thanks @jacob-votava!
